### PR TITLE
update documentation for Monocular 1.0

### PR DESCRIPTION
**Pull request by prydonius**
- update goal of project to support Helm Hub
- update developer documentation for telepresence-based development environment
- remove in-cluster configuration/documentation
- add note to readme about alternatives for in-cluster app management

Signed-off-by: Adnan Abdulhussein <adnan@bitnami.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,24 +2,20 @@
 [![Build
 Status](https://travis-ci.org/helm/monocular.svg?branch=master)](https://travis-ci.org/helm/monocular)
 
-Monocular is web-based UI for managing Kubernetes applications packaged as Helm
-Charts. It allows you to search and discover available charts from multiple
-repositories, and install them in your cluster with one click.
+Monocular is a web-based UI and API that enables the search and discovery of
+charts from multiple Helm Chart repositories. It is the codebase that powers the
```
Comment (migmartri) [inline README.md]: powers? will power?

File: README.md [hunk 2]
```diff
@@ -2,24 +2,20 @@
 [![Build
 Status](https://travis-ci.org/helm/monocular.svg?branch=master)](https://travis-ci.org/helm/monocular)
 
-Monocular is web-based UI for managing Kubernetes applications packaged as Helm
-Charts. It allows you to search and discover available charts from multiple
-repositories, and install them in your cluster with one click.
+Monocular is a web-based UI and API that enables the search and discovery of
```
Comment (migmartri) [inline README.md]: Maybe a higher level description would make more sense, i.e `Monocular is a web-based **application** that enables the search and discovery of...`

File: README.md [hunk 3]
```diff
@@ -53,6 +49,19 @@ Read more on how to deploy Monocular [here](chart/monocular/README.md).
 - [Deployment](chart/monocular/README.md)
 - [Development](docs/development.md)
 
+## Looking for an in-cluster Application management UI?
+
+To focus on the CNCF Helm Hub requirements, in-cluster features have been
+removed from Monocular 1.0 and above. We believe that providing a good solution
+for deploying and managing apps in-cluster is an orthogonal user experience to a
+public search and discovery site. There is other tooling that can support this
```
Comment (migmartri) [inline README.md]: other tooling vs other tool?

File: README.md [hunk 4]
```diff
@@ -53,6 +49,19 @@ Read more on how to deploy Monocular [here](chart/monocular/README.md).
...
+usecase better (e.g. [Kubeapps](https://github.com/kubeapps/kubeapps) or [RedHat
+Automation
+Broker](https://blog.openshift.com/automation-broker-discovering-helm-charts/)).
+
+Monocular v0.7.3 includes in-cluster features and can still be installed and
```
Comment (migmartri) [inline README.md]: Would it make sense to link to the github tag?

File: docs/development.md [hunk 5]
```diff
@@ -4,40 +4,55 @@ This guide explains how to set up your environment for developing on Helm and Ti
...
-* A Kubernetes cluster with Helm/Tiller installed (optional)
+* A Kubernetes cluster with Helm/Tiller installed
+* Telepresence 0.75 or later
 * kubectl 1.2 or later (optional)
 * Git
 
 ## Running Monocular
 
-We use [Docker Compose](https://docs.docker.com/compose/) to orchestrate the UI frontend and API backend containers for local development. The simplest way to get started is:
+We develop Monocular in a Kubernetes environment. Minikube can be used to run a
```
Comment (migmartri) [inline docs/development.md]: Maybe you can give some context on why k8s is needed? For example mentioning that the chart import job uses native k8s cronjobs

File: README.md (hunk 1 above)
Comment (prydonius) [reply README.md]: If we say "will power" we'll need to go and update this later, I think it's okay to just say powers here.

File: README.md (hunk 3 above)
Comment (prydonius) [reply README.md]: hmm any particular reason to use tools over tooling?

Comment (prydonius) [reply README.md]: Ah I see, no we also mention the RedHat Automation Broker

**Comment by migmartri** [comment]
lgtm

**Pull request merged**